tests/fast_reboot_dump_dbs/APPL_DB.json

```diff
@@ -0,0 +1,10 @@
+{
+  "LAG_MEMBER_TABLE:PortChannel0001:Ethernet128": {
```

Comment:
could you please add a comment of what is the below configuration stands for and will be used for? you can do it also in the test case itself

Reply:
Added a comment above the test case.